lib/assessment.rb

```diff
@@ -35,6 +36,7 @@ def every_even(numbers)
   # entries from the words enumerable capitalized.
   def transform(words)
     # Your code goes here
+    words.upcase.to_a
```

Comment:
Not the right tool for this. When you perform a transformation across athe elements of a set, you want to use `.map`